src/submission/models.py

```diff
@@ -471,6 +477,32 @@ class Article(models.Model):
     class Meta:
         ordering = ('-date_published', 'title')
 
+    @property
+    def how_to_cite(self):
```

Comment:
Would it be easier/neater to use a template and render_to_string for this as we're generating HTML?